### PR TITLE
Add login page layout

### PR DESCRIPTION
**Pull request by frgiovanna**
## Motivation

This PR adds the layout of the Login page.

## Screenshots

Screenshot 2024-11-29 at 11 14 46

### CODE REVIEW

File: src/shared/theme/theme.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        light: '#FFFFFF',
+        dark: '#16408F',
+      },
+    },
+    background: {
+      light: '#FFF8EE',
+      sand: '#FFE6C5',
+      gradient: 'linear-gradient(270deg, #FF901A 0%, #FB6203 100%)',
+    },
+  },
```
Comment (frgiovanna) [inline src/shared/theme/theme.ts:18]: @Wendlereis could you help me with the organization of those colors? I tried to separate by the semantics but I've been using some colors in more than one context and I don't know if it hindered more than it helped

File: src/shared/components/Card/styles.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  box-shadow: ${theme.shadow};
+
+  ${({ position }) =>
+    position === 'bottom'
+      ? css`
+          border-radius: 40px 40px 0px 0px;
+        `
+      : css`
+          border-radius: 0px 0px 40px 40px;
+        `}
```
Comment (Wendlereis) [inline src/shared/components/Card/styles.ts:19]: **Suggestion:** How about this? Does it work with emotion or we always need the `css`?
```suggestion
  border-radius: ${({ position }) => position === 'bottom' 
    ? '40px 40px 0px 0px' 
    : '0px 0px 40px 40px;'}
```

File: src/shared/components/Typography/Typography.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { theme } from '../../theme';
+import { Body1, Body2, H1, H2, H3 } from './styles';
+import { TypographyProps } from './types';
+
+export const Typography = ({
```
Comment (Wendlereis) [inline src/shared/components/Typography/Typography.tsx:5]: **Question:** I've noticed that for Button, Input, and Card you used named function and here is a function expression. What is your preferred pattern for this case?

File: src/shared/components/Typography/Typography.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  children,
+  color = theme.colors.text.body,
+  variant = 'Body1',
+  textAlign = 'left',
+}: TypographyProps) => {
+  const setup = {
```
Comment (Wendlereis) [inline src/shared/components/Typography/Typography.tsx:11]: **Suggestion:** How about
```suggestion
  const buttonVariants = {
```

File: src/shared/components/Typography/styles.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+import { StyledTypographyProps } from './types';
+import { css } from '@emotion/react';
+
+export const H1 = styled.h1<StyledTypographyProps>`
+  font-size: 40px;
+  font-weight: 600;
+  margin: 0;
```
Comment (Wendlereis) [inline src/shared/components/Typography/styles.ts:8]: **Suggestion:** I see a lot repetition for `margin: 0`. How about moving it to `src/index.css` file. 

```css
h1,
h2,
h3,
p {
  margin: 0;
}
```

File: src/shared/components/Typography/styles.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+`;
+
+export const H2 = styled.h2<StyledTypographyProps>`
+  font-size: 32px;
+  font-weight: 600;
+  margin: 0;
+  ${({ color, textAlign }) => css`
+    color: ${color};
+    text-align: ${textAlign};
+  `}
```
Comment (Wendlereis) [inline src/shared/components/Typography/styles.ts:22]: **Question:** Is it possible to extract it to a variable?

File: src/shared/theme/theme.ts (hunk 1 above)
Comment (Wendlereis) [reply src/shared/theme/theme.ts:18]: This granular approach is great! I'll go with it

File: src/shared/components/Typography/styles.ts (hunk 6 above)
Comment (frgiovanna) [reply src/shared/components/Typography/styles.ts:22]: I did it as a function, let me know if you liked it.

**Review by Wendlereis — APPROVED** [review]
🙌

**Pull request merged**